cmd/api/src/model/filter.go

```diff
@@ -165,6 +170,9 @@ func (s QueryParameterFilterMap) BuildSQLFilter() (SQLFilter, error) {
 				predicate = EqualsSymbol
 			case NotEquals:
 				predicate = NotEqualsSymbol
+			case ApproximatelyEquals:
+				predicate = ApproximatelyEqualSymbol
+				filter.Value = fmt.Sprintf("%%%s%%", filter.Value)
```

Comment:
I tested adding `%` in Bruno and adding `%` acts as a an empty character, see screenshots